nautobot/ipam/tests/test_filters.py

```diff
@@ -571,8 +567,22 @@ def setUpTestData(cls):
         )
 
     def test_search(self):
-        ipv4_address = self.queryset.ip_family(4).exclude(dns_name="").exclude(dns_name__isnull=True).first()
-        ipv6_address = self.queryset.ip_family(6).exclude(dns_name="").exclude(dns_name__isnull=True).first()
+        ipv4_address = (
+            self.queryset.ip_family(4)
+            .exclude(dns_name="")
+            .exclude(dns_name__isnull=True)
+            .order_by("last_updated")
+            .last()
+        )
+        # This is not getting ipv6 address with mask: `2001:db8::1/64`` consistently
+        # instead it is getting `8ca:d203:559e:5211:e0fc:3ae8:72:623e/128`` so last test f"{ipv6_hextets[0]}::" would fail.
+        ipv6_address = (
+            self.queryset.ip_family(6)
+            .exclude(dns_name="")
+            .exclude(dns_name__isnull=True)
+            .order_by("last_updated")
+            .last()
+        )
```

Comment:
Feels kinda gross... if we really need specific IPAddress instances that were created in `setUp`, let's just refer to them directly instead of all this complex filtering.